# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/mesa-demos/hotfix-uint.patch.musl
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- mesa-demos-9.0.0/src/vulkan/wsi/wayland.c.vanilla	2023-07-28 12:25:02.152790066 +0200
+++ mesa-demos-9.0.0/src/vulkan/wsi/wayland.c	2023-07-28 12:26:01.777787970 +0200
@@ -48,6 +48,8 @@
 #include <xkbcommon/xkbcommon-keysyms.h>
 #include <xkbcommon/xkbcommon.h>
 
+typedef unsigned int uint;
+
 #include "wsi.h"
 
 static struct wsi_callbacks wsi_callbacks;
